insights/parsers/systemctl_show.py

```diff
@@ -107,3 +110,101 @@ class SystemctlShowMariaDB(SystemctlShow):
         network.target -.mount systemd-journald.socket tmp.mount basic.target syslog.target system.slice
     """
     pass
+
+
+@parser(systemctl_pulp_workers)
+class SystemctlShowPulpWorkers(SystemctlShow):
+
+    """Class for ``systemctl show pulp_workers``
+
+       Typical output of ``/bin/systemctl show pulp_workers`` command is::
+
+        Type=oneshot
+        Restart=no
+        NotifyAccess=none
+        RestartUSec=100ms
+        TimeoutStartUSec=0
+        TimeoutStopUSec=1min 30s
+        WatchdogUSec=0
+        WatchdogTimestampMonotonic=0
+        ExecMainStartTimestamp=Thu 2018-01-11 14:22:33 CST
+        ExecMainStartTimestampMonotonic=105521850
+        ExecMainExitTimestamp=Thu 2018-01-11 14:22:33 CST
+        ExecMainExitTimestampMonotonic=105593405
+        ExecStart={ path=/usr/libexec/pulp-manage-workers ; argv[]=/usr/libexec/pulp-manage-workers start ; ignore_err
+        ExecStop={ path=/usr/libexec/pulp-manage-workers ; argv[]=/usr/libexec/pulp-manage-workers stop ; ignore_error
+        Slice=system.slice
+        After=systemd-journald.socket system.slice network.target basic.target
+        ...
+
+    Examples:
+        >>> SrvPulpworkers = shared[SystemctlPulpWorkers]
```

Comment:
Sorry for miss this one in the last review.

Since `shared` does not exist anymore in 3.x, better change it to something more consistent with 3.x.  Please refer to `tomcat_xml.py` or `jboss_versoin.py`

Reply:
I can not find tomcat_xml.py or jboss_versoin.py files under parsers, where are they

Reply:
You should update your local repo to the latest at first:

~~~
# git checkout master
# git pull origin master
~~~